src/poetry/console/commands/init.py

```diff
@@ -154,7 +154,14 @@ def handle(self) -> int:
         python = self.option("python")
         if not python:
             current_env = SystemEnv(Path(sys.executable))
-            default_python = "^" + ".".join(
+
+            from poetry.config.config import Config
+
+            config = Config.create()
+            prefix = config.get("default-python-prefix")
+            if prefix is None:
+                prefix = "^"
```

Comment:
I would prefer to see it as a method. `handle` methods are already pretty big, let's avoid putting even more into them.

Reply:
OK

Reply:
I'm pushing a new commit that does this in its own method.